src/Components/SigTable/SigTable.js

```diff
@@ -94,6 +96,13 @@ const SigTable = () => {
                 { label: intl.formatMessage(messages.matched), value: 'true' },
                 { label: intl.formatMessage(messages.notMatched), value: 'false' }
             ]
+        },
+        sigs_enabled: {
+            type: 'radio', title: intl.formatMessage(messages.signaturesEnabledFilter), urlParam: 'sigs_enabled', values: [
```

Comment:
In the mockup the filter has checkboxes instead of radio buttons. Is it intentional using radio buttons, or should we change this in order to match the mockups?

Reply:
I intentionally used radio buttons because it was easier for me to implement :smile: (and easier to pass the radio button result to the backend).  I'll put it out there for people to try and if they feel strongly about having checkboxes instead, then I'll change it.